items/blind.lua

```diff
@@ -675,6 +675,40 @@ local scorch = {
 		end
 	end,
 }
+-- +0.25X blind requirements
+-- for every $5 you have when selected
+local greed = {
+	dependencies = {
+		items = {
+			"set_cry_blind",
+		},
+	},
+	object_type = "Blind",
+	name = "cry-greed",
+	key = "greed",
+	pos = { x = 0, y = 0 }, -- use Tax as placeholder icon
+	boss = {
```

Comment:
Please move the magic numbers for money factor and blind change into an extra table. 
```suggestion
    config = {
        extra = {
            money_factor = 5,
            blind_mod = 0.25,
        },
    },
	boss = {
```

If you don't want these to be affected by misprintize (the thing that modifies default joker values), put them in an immutable table.

```suggestion
    config = {
        immutable = {
            money_factor = 5,
            blind_mod = 0.25,
        },
    },
	boss = {
```

Reply:
uhh, this is a boss blind, not a joker

Reply:
Understood, I got too comment happy. 

Are we able to use config within boss blinds? 

I really want to rid all magic numbers

Reply:
it is possible but currently it won't work with obsidian orb if it's done that way due to how obsidian orb is coded

Reply:
So currently, does obisidan orb need to updated everytime a new blind is added then?